src/app/app.module.ts

```diff
@@ -22,13 +22,15 @@ import {MatNativeDateModule} from '@angular/material/core';
 import {MatCheckboxModule} from '@angular/material/checkbox';
 import { CompaniesComponent } from './companies/companies.component';
 import { PdfViewerDialogComponent } from './pdf-viewer-dialog/pdf-viewer-dialog.component';
+import { ProjectComponentComponent } from './project-component/project-component.component';
```

Comment:
Please change the name of Project files to Project.component.html/.ts/.css/.spec.ts.
Lets follow same naming convention as we are using in our same file, b'se two times "component" is not making sense. Please change the name of ProjectComponentComponent to ProjectComponent .

Reply:
You haven't changed  ProjectComponentComponent to ProjectComponent . Please take care of comment from next time or discuss if you don't agree with comment. I'm approving PR for now.